src/components/settings/SettingsLMSTab/ErrorReporting/DownloadCsvButton.jsx

```diff
@@ -7,7 +7,7 @@ import {
 import { Download, Check } from '@edx/paragon/icons';
 import { isStrictlyArray } from './utils';
 
-const DownloadCsvButton = ({ data }) => {
+const DownloadCsvButton = ({ data, testId }) => {
```

Comment:
Are we anticipating using this component in multiple places?

Reply:
Yes, we're using one on both tabs so this is an easy and accessible way to differentiate them in testing!